source/_docs/energy/individual-devices.markdown

```diff
@@ -24,6 +24,9 @@ We recommend the Shelly brand because they have a local API that updates Home As
 - [Shelly 2.5](https://shop.shelly.cloud/shelly-2.5-ce-ul-wifi-smart-home-automation?tracking=A7FsiPIfUWsFpnfKHa8SRyUYLXjr2hPq)
  {% endcomment %}
 
+### Devices with current (W) sensors
```

Comment:
```suggestion
### Devices with power (W) sensors
```

Reply:
Maybe "with current/power (W) sensors", or something like that? I'm pretty sure on my searches I saw other "current" stuff; besides that, my LG AC's sensor is `ac_bedroom_energy_current`.

Reply:
No power is the correct term. A current sensor is measured in Amperage (A) To calculate the power draw (W) You would need to do V * A.

Reply:
Funny. So I guess I should report a misnomer to the LG package hahaha

Reply:
`ac_bedroom_energy_current` sounds like it should give values with **kWh** unit of measurement.